### DISCUSSION TITLE
Embedding a game to webpage or website

### DISCUSSION THREAD

**Discussion by HeeteshSimon** [EddieHubCommunity/support#241] [Q&A]
Hey, I have created Tetris using Pygame. I want to deploy that game on a website or a web page. I not sure how to start with it. Any kind of help or suggestion will be much appreciated.

**Comment by eddiejaoude** [comment]
If it is python it will need to be server side.

There was a discussion about python hosting here https://github.com/EddieJaoudeCommunity/support/discussions/135

**Comment by AllanRegush** [comment]
I don't think you can do that directly. You can try with http://skulpt.org/ or https://py3.codeskulptor.org/

**Comment by akshayprog** [comment]
read this topic on stackoverflow - [https://stackoverflow.com/questions/56350007/how-to-run-a-python-pygame-file-on-a-django-website-without-using-path?noredirect=1&lq=1](url)

**Comment by mikeysan** [comment]
@HeeteshSimon Did you find a solution to your question yet? I found this interesting [brython](https://github.com/brython-dev/brython) Perhaps it might help.
